NAMESPACE

```diff
@@ -30,6 +30,13 @@ S3method(format,DescriptionAuthorsAtR)
 S3method(format,DescriptionCollate)
 S3method(format,DescriptionDependencyList)
 S3method(format,DescriptionField)
+S3method(to_latex,DescriptionAuthorsAtR)
+S3method(to_latex,DescriptionCollate)
+S3method(to_latex,DescriptionField)
+S3method(to_latex,DescriptionURL)
+S3method(to_latex,DescriptionURLList)
+S3method(to_latex,character)
+S3method(to_latex,person)
```

Comment:
Hmmm, I would not necessarily export all these. I think we only need one method for the whole `Description`, the rest is implementation details, and internal.